llvm/test/Transforms/GlobalDCE/used.conditional5.ll

```diff
@@ -1,17 +0,0 @@
-; RUN: opt -S -passes=globaldce < %s | FileCheck %s
```

Comment:
This test tested what happens when the metadata contains null dependencies, which is now impossible.